src/components/FetchApi.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getGames } from '../redux/gameSlice';
+import image from '../images/cryptocurrencyimage.jpeg';
+
+function FetchApi() {
+  const dispatch = useDispatch();
+  const allGames = useSelector((state) => state.game);
+
+  useEffect(() => {
+    dispatch(getGames());
+  }, []);
+  return (
+    <div className="flex bg-pink-100 flex-col text-white">
+      <div className="flex items-center justify-center space-x-2 h-32">
+        <img src={image} alt="crypto" className="w-1/2" />
+        <div>
+          <h1 className="font-extrabold text-2xl">Crypto Exhange</h1>
+          <p className="font-semibold text-lg">
+            {allGames.length}
+            {' '}
+            crypto
+          </p>
+        </div>
+      </div>
+      <div className="text-start pl-5 bg-pink-200 text-xl font-medium">
+        Stats
+      </div>
+      <ul className="grid grid-cols-2">
+        {allGames.map((game, id) => (
+          <Link to={`details/${id}`} key={game.id}>
+            <li className="flex flex-col bg-pink-300 items-center justify-center h-28 p-6">
+              <img className="w-2/5 self-start" src={game.image} alt="crypto" />
+              <div className="self-end">
+                <p className="font-bold text-lg">{game.name}</p>
+                <p className="font-semibold">
+                  $
+                  {game.current_price}
+                </p>
+              </div>
+            </li>
+          </Link>
+        ))}
+      </ul>
```

Comment:
- [ ] [REQUIRED] Great job with the design so far, however, you need to do some finishes to the homepage UI. Ensure that each `li` element is properly separated. You have placed them in grids as expected but each element needs more padding and margins at it currently looks clumsy. Note that the elements are separated with similar shades of colors which grants it a better UI. Please take a look at the spacing of your design below and compare it with the template provided. Kindly fix this for a better UI and to satisfy the project requirement.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-07-29 13-18-09](https://user-images.githubusercontent.com/58692590/181757089-eb7fd581-ad33-4ab3-aa57-72ed1dc721dd.png) | ![Screenshot from 2022-07-29 13-18-00](https://user-images.githubusercontent.com/58692590/181757420-d4d4b16d-2d8b-497e-82fc-f66f851e93b0.png)